src/transformers/training_args.py

```diff
@@ -113,6 +113,9 @@ class TrainingArguments:
             at the next training step under the keyword argument ``mems``.
         run_name (:obj:`str`, `optional`):
             A descriptor for the run. Notably used for wandb logging.
+        disable_tqdm (:obj:`bool`, `optional`):
```

Comment:
Adding this since all the progressbars while in hyperparameter search are overwhelming the output.

Reply:
Will probably change the default to `None` once the logging PR is merged